vmdb/lib/miq_automation_engine/service_models/miq_ae_service_service.rb

```diff
@@ -118,5 +118,12 @@ def group=(group)
         @object.save
       end
     end
+
+    def add_resource(rsc, options = {})
```

Comment:
Exposing add_resource here allows anything to be connected to a service which is not something we want at the moment.  I would prefer this be ```add_to_service``` on the object you want to connect.  We only need to connect the stack post-provisioning, right?
Example here: https://github.com/ManageIQ/manageiq/blob/master/vmdb/lib/miq_automation_engine/service_models/miq_ae_service_vm.rb#L3